coroutines-extra/src/main/scala/AsyncAwait.scala

```diff
@@ -0,0 +1,101 @@
+package org.coroutines.extra
```

Comment:
The package names looks good! But we should be consistent with relative paths to files - if the file has a package `x.y.z`, then it should be inside (in this case) `coroutines-extra/src/main/scala/x/y/z`.